src/features/venue/components/VenueOffers/VenueOffersNew.tsx

```diff
@@ -64,7 +68,12 @@ export function VenueOffersNew({ venueId, layout = 'two-items' }: Readonly<Props
     [position]
   )
 
-  const showSeeMore = nbHits > hits.length
+  const shouldDisplayGtlPlaylist =
+    [VenueTypeCodeKey.DISTRIBUTION_STORE, VenueTypeCodeKey.BOOKSTORE].includes(
+      venue?.venueTypeCode as VenueTypeCode
+    ) && Boolean(playlists?.length)
```

Comment:
Je te conseille de faire comme ça, ça permettra d'enlever le `?` ligne 116
```suggestion
    ) && !!playlists?.length
```